app/src/main/java/dev/arkbuilders/arkmemo/ui/fragments/ArkMediaPlayerFragment.kt

```diff
@@ -34,10 +34,8 @@ class ArkMediaPlayerFragment: BaseEditNoteFragment() {
 
     private val arkMediaPlayerViewModel: ArkMediaPlayerViewModel by viewModels()
 
-    private lateinit var seekBar: SeekBar
+//    private lateinit var seekBar: SeekBar
```

Comment:
We should not forget to remove it before merging.

Reply:
@kirillt Sure. I'm gonna remove all commented code.